torchci/components/WorkflowBox.tsx

```diff
@@ -87,6 +111,7 @@ export default function WorkflowBox({
   const anchorName = encodeURIComponent(workflowName.toLowerCase());
 
   const { artifacts, error } = useArtifacts(workflowId);
+  const [artifactsToShow, setArtifactsToShow] = useState(new Set<string>());
```

Comment:
Nice, using a set here would keep the `useState` hook a constant.  On the other hand, I think you could test having more workflows by using the `run more tests` checkbox (although I think that's probably a rare case that I don't mind refreshing the page)

Reply:
So far, I'm using your preview page and haven't seen any errors.